README.md

```diff
@@ -125,7 +125,6 @@ A full list of all contributors can be found [here][contributors].
 [changelog]: https://materialsproject.github.io/atomate2/user/changelog.html
 [installation]: https://materialsproject.github.io/atomate2/user/install.html
 [contributing]: https://materialsproject.github.io/atomate2/user/contributing.html
-[contributors]: https://materialsproject.github.io/atomate2/user/contributors.html
```

Comment:
Why remove the contributors page?

Reply:
Oops, that was by mistake. Got a bit late last night.